src/event-db/setup/graphql-setup.sql

```diff
@@ -60,13 +60,10 @@
 \echo VARIABLES:
 \echo -> dbName ....................... = :dbName
 \echo -> dbUser ....................... = :dbUser
-\echo -> adminRolePw / $ADMIN_ROLE_PW . = :adminRolePw
-\echo -> anonRolePw / $ANON_ROLE_PW ... = :anonRolePw
 \echo -> adminUserFirstName ........... = :adminUserFirstName
 \echo -> adminUserLastName ............ = :adminUserLastName
 \echo -> adminUserAbout ............... = :adminUserAbout
 \echo -> adminUserEmail ............... = :adminUserEmail
-\echo -> adminUserPw / $ADMIN_USER_PW . = :adminUserPw
 
```

Comment:
I know your deleting these for security.  Can we instead just put `***` as the value.
The reason being that I like this publishing all possible options from a self documentation perspective.